### DISCUSSION TITLE
Teslamate does not work after installing portainer

### DISCUSSION THREAD

**Discussion by ghost** [teslamate-org/teslamate#1094] [Q&A]
Hello

I hab the docker installation on my raspi 4 and decided to use portainer for the docker management. After that teslamate can not retrieve data from the car anymore. the dashboard is reachable and still working fine, but i do not have data since yesterday evening. Can anyone give an advice? It is still possible to communicate outside the cintainer (ping goole, etc.)

Thank you and Regards,
Marc

**Comment by DrMichael** [comment]
What is in the logs of the teslamate and portainer containers?
